MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,349 @@
+
+```yaml
+logsCollection:
+  extraFileLogs:
+  filelog/kube-audit:
+    include: [/var/log/kube-apiserver-audit.log]
+    start_at: beginning
+    include_file_path: true
+    include_file_name: false
+    resource:
+      service.name: /var/log/kube-apiserver-audit.log
+      host.name: 'EXPR(env("K8S_NODE_NAME"))'
+      com.splunk.sourcetype: kube:apiserver-audit
+```
+
+Use the `kube-audit` keyword to continue reading from the translated checkpoint data.
+
+### Translating custom configurations from SCK to Splunk OpenTelemetry Collector for Kubernetes for metrics
+
+#### Limiting or increasing application resources"
+
+If you are using the `resources` option in SCK to limit/increase the CPU and memory usage in SCK, you can do the same using the `resources` option in `clusterReceiver` section in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod scheduling configurations
+
+If you are using any pod scheduling operations such as `nodeSelector`, `affinity`, and `tolerations` in SCK, you can use the `clusterReceiver.nodeSelector`, `clusterReceiver.affinity` and `clusterReceiver.tolerations` options in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod security context
+
+If you are using the `podSecurityContext` option to set a pod security policy, you can use the `clusterReceiver.securityContext` option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom annotations
+
+If you are using the `customMetadataAnnotations` option to set a custom annotation fields in your logs in SCK, you can use the `extraAttributes.fromAnnotations` option in Splunk OpenTelemetry Collector for Kubernetes.
+
+## (Optional) Step 2: Delete SCK Deployment
+
+To delete the SCK deployment, find the name of the deployment using the `helm ls` command.
+
+* **If you want to delete only logs and metrics**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete only logs**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete your entire SCK deployment (logs, objects and metrics), run the following command**
+
+  * helm delete local-k8s
+
+## Step 3: Installing Splunk OpenTelemetry Collector for Kubernetes
+
+* See [How to install](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/README.md#how-to-install) for the instructions.
+
+## Step 4: Check data in Splunk
+
+* Check the logs index to see if you are receiving logs from your Kubernetes cluster
+  * index="Your logs index"
+* Check the metrics index to see if you are receiving logs from your Kubernetes cluster
+  * | mcatalog values(metric_name) WHERE index="Your metrics index"
+
+## Step 5: Delete SCK Deployment if not done in Step 2
```

Comment:
A header without content seems wrong. This one probably not necessary and can be removed?